packages/rancher-vsphere/rancher-vsphere-csi/generated-changes/patch/templates/node/daemonset.yaml.patch

```diff
@@ -0,0 +1,23 @@
+--- charts-original/templates/node/daemonset.yaml
++++ charts/templates/node/daemonset.yaml
+@@ -19,7 +19,7 @@
+         role: vsphere-csi
+     spec:
+       {{- if .Values.csiNode.nodeSelector }}
+-      nodeSelector:
++      nodeSelector: {{ include "linux-node-selector" . | nindent 8 }}
```

Comment:
if nodeSelectors are not supplied, this code worked by using affinity.

If nodeSelectors are supplied, this code was broken on Windows clusters since default node-selectors are not applied for Windows.